proposed/2021/NuGet.orgTFMs.md

```diff
@@ -0,0 +1,96 @@
+# NuGet Target Frameworks on Package Details Page 
+
+- Author Name (https://github.com/jcjiang)
+- Start Date (2021-02-16)
+- GitHub Issue (https://github.com/NuGet/NuGetGallery/issues/8387)
+- GitHub PR ()
+
+## Summary
+
+When a developer opens the package details page or tab on NuGet.org or NuGet rooling, they will see what frameworks the package can support. 
```

Comment:
```suggestion
When a developer opens the package details page or tab on NuGet.org or NuGet tooling, they will see what frameworks the package can support. 
```

Reply:
I think tooling shouldn't be here. I stripped out most of them and missed this one.

Reply:
Removed.